tests/Makefile: Replace hardcoded "i386" with "dpkg --print-architecture"
so that the test runs on other architectures.

diff -Nru adequate-0.9/tests/Makefile adequate-0.9ubuntu1/tests/Makefile
--- adequate-0.9/tests/Makefile	2013-04-04 21:49:00.000000000 +0200
+++ adequate-0.9ubuntu1/tests/Makefile	2013-12-18 08:05:28.000000000 +0100
@@ -1,18 +1,22 @@
 .PHONY: all
-all: adequate-testpkg_1_i386.changes
+
+ARCH=$(shell dpkg --print-architecture)
+TEST_CHANGES=adequate-testpkg_1_${ARCH}.changes
+
+all: ${TEST_CHANGES}
 	$(info Run "make run" as root.)
 	$(info Alternatively, if you have user-mode-linux installed, you can try "make run-uml" as normal user.)
 	$(info )@:
 
-adequate-testpkg_1_i386.changes:
+${TEST_CHANGES}:
 	cd testpkg && dpkg-buildpackage -b -us -uc
 
 .PHONY: run
-run: adequate-testpkg_1_i386.changes
+run: ${TEST_CHANGES}
 	./run-tests $(<)
 
 .PHONY: run-uml
-run-uml: adequate-testpkg_1_i386.changes
+run-uml: ${TEST_CHANGES}
 	fallocate -l 1G uml-guest-swap
 	setsid linux init=$(CURDIR)/uml-guest-init \
 		$(shell printf '%s' '$(CURDIR)' | base64 -w0 | tr = _) \
